_includes/schemas/shipments_request.json

```diff
@@ -194,6 +194,10 @@
           "type": "number",
           "description": "additional custom fees to be payed"
         },
+        "movement_reference_number": {
```

Comment:
I think all (or most) other attributes are sorted alphabetically. Could we do that here as well?
If for any reason it makes more sense to you to keep it here I don't mind. I just noticed the order

Reply:
Fixed: https://github.com/shipcloud/shipcloud-developer-portal/pull/315/commits/12079d527577b65035ad7cd23fb9803cb0a94dd6

Reply:
Could you do that for the other files as well :D